### PR TITLE
Upload Files

### PR DESCRIPTION
**Pull request by AmrNabel**

### CODE REVIEW

**Review by sahdasamier — CHANGES_REQUESTED** [review]
there are aproplem in sizes of boxes , it is so big

File: src/UploadFiles/UploadFiles.tsx [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+          <TextField
+            label="Email"
+            sx={{
+              width: "386px",
+              padding: "14px 126px 13 px 13px",
+              mt: "10px",
+              height: "48px",
+            }}
+          />
+          <Button
```
Comment (sahdasamier) [inline src/UploadFiles/UploadFiles.tsx:169]: this button, when you hover over it , it being like the background and totally adding so please add hovering color

**Review by sahdasamier — CHANGES_REQUESTED** [review]

File: src/UploadFiles/UploadFiles.tsx
[the remark this answers is not in the document]
Comment (AmrNabel) [reply src/UploadFiles/UploadFiles.tsx:1]: I did

File: src/UploadFiles/UploadFiles.tsx [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+        sx={{
+          width: "787px",
+          height: "221px",
+          m: "0 auto 55px 64px",
+          border: "1px solid #0A0A0A33",
+          borderRadius: "5px",
+          padding: "15px 16px 15px 41px",
+        }}
+      >
+        <Typography
```
[the remark this answers is not in the document]
Comment (AmrNabel) [reply src/UploadFiles/UploadFiles.tsx:134]: I don't see any missing button in the component compared to the given design

**Review by sahdasamier — CHANGES_REQUESTED** [review]
great job Amr, but please adjust the alert message like what I said

**Review by sahdasamier — APPROVED** [review]

**Pull request merged**